packages/shared/src/lib/boot.ts

```diff
@@ -78,5 +85,20 @@ export async function getBootData(app: string, url?: string): Promise<Boot> {
     credentials: 'include',
     headers: { app, 'Content-Type': 'application/json' },
   });
-  return res.json();
+  const result = await res.json();
+
+  try {
+    const features = await decrypt(
```

Comment:
This mean we will decrypt it every time we fetch. Before we decrypted and initialized GB only once, what behavior do we prefer?

Reply:
Yes we do decrypt each time, thats why we did performance tests to see the impact.

GB is still initalized only once.